requirements/app/base.txt

```diff
@@ -13,3 +13,4 @@ inquirer>=2.10.0
 psutil<5.9.4
 click<=8.1.3
 lightning_api_access>=0.0.3
+aiohttp<=3.8.3
```

Comment:
what is the min version?

Reply:
Not sure at the moment. I'll bisect versions and set the lower bound here.